src/index.css

```diff
@@ -38,10 +38,27 @@ h2 {
   padding: 0
 }
 
-button {
+.ACES {
   position: absolute;
   top: 1rem;
   right: 4rem;
   padding: 4px 30px;
   border-radius: 4px;
+}
+
+.Olin {
```

Comment:
This css for Olin button need to make more css for other buttons use this as an example. Currently this one is places at a random spot.